runtime/compiler/env/VMJ9.cpp

```diff
@@ -1565,15 +1565,13 @@ UDATA TR_J9VMBase::getHighTenureAddress()
 
 bool TR_J9VMBase::generateCompressedPointers()
    {
-   return TR::Options::useCompressedPointers();
+   return TR::Compiler->om.compressObjectReferences();
    }
 
 
 bool TR_J9VMBase::generateCompressedLockWord()
    {
-   if (sizeof(j9objectmonitor_t) == 4)
-      return true;
-   return false;
+   return TR::Compiler->om.compressObjectReferences();
```

Comment:
The old code's check `(sizeof(j9objectmonitor_t) == 4)` covered 32 bit.  Is this function only called on 64 bit or have you tested 32 bit already?

Reply:
Good point - this would return true for 32-bit before and now would not. 32-bit works, so I suspect it's not called, but I'll update it to do the reference size check instead.